elastic/eui
# feat: setup new docs

# Summary

This PR configures the docusaurus and adds sample documentation files. Note that it targets the new `feat/new-docs` feature branch and is meant to be used primarily for testing.

# QA

1. Checkout this branch locally and `cd` into the `website` directory
2. Run `yarn` and `yarn start`
3. Confirm the dev server is loading and http://localhost:3000/docs/guidelines/getting-started renders the sample documentation page
